Essentials/src/main/resources/messages.properties

```
@@ -249,6 +263,12 @@ discordErrorNoGuild=Invalid or missing server ID! Please follow the tutorial in
 discordErrorNoGuildSize=Your bot is not in any servers! Please follow the tutorial in the config in order to setup the plugin.
 discordErrorNoPerms=Your bot cannot see or talk in any channel! Please make sure your bot has read and write permissions in all channels you wish to use.
 discordErrorNoToken=No token provided! Please follow the tutorial in the config in order to setup the plugin.
+discordLinkLinked=\u00a76To link this minecraft account to a discord account, type \u00a7c{0} \u00a76in discord in order to link this account.
+discordLinkLoginKick=\u00a76You must link your discord account before you can join this server.\n\u00a76To link your discord account, type\:\n\u00a7c{0}\n\u00a76in this server''s discord server\:\n\u00a7c{1}
+discordLinkLoginPrompt=\u00a76You must link your discord account before you can move/chat/interact with this server. To link your discord account, type \u00a7c{0} \u00a76in this server''s discord server\: \u00a7c{1}
+discordLinkNoAccount=\u00a76You do not currently have a discord account linked to this minecraft account.
+discordLinkPending=\u00a76You already have a link code. Type \u00a7c{0} \u00a76in discord in order to link this account.
+discordLinkUnlinked=\u00a76Unlinked this minecraft account from all associated discord accounts.
```

```suggestion
discordLinkLinked=\u00a76To link your Minecraft account to Discord, type \u00a7c{0} \u00a76in the Discord server.
discordLinkLoginKick=\u00a76You must link your Discord account before you can join this server.\n\u00a76To link your Minecraft account to Discord, type\:\n\u00a7c{0}\n\u00a76in this server''s Discord server\:\n\u00a7c{1}
discordLinkLoginPrompt=\u00a76You must link your Discord account before you can move, chat on or interact with this server. To link your Minecraft account to Discord, type \u00a7c{0} \u00a76in this server''s Discord server\: \u00a7c{1}
discordLinkNoAccount=\u00a76You do not currently have a Discord account linked to your Minecraft account.
discordLinkPending=\u00a76You already have a link code. To complete linking your Minecraft account to Discord, type \u00a7c{0} \u00a76in the Discord server.
discordLinkUnlinked=\u00a76Unlinked your Minecraft account from all associated discord accounts. TODO
```

TODO check one-to-many/one-to-one/many-to-many again